src/controllers/product.js

```diff
@@ -235,11 +235,40 @@ const orderRequest = async (req, res) => {
   }
 };
 
+const seeRequesters = async (req, res) => {
```

Comment:
Adnan we can change the naming getRequesters? maybe?

Reply:
You're absolutely right! I'll change all of them asap.